apps/ui/src/components/ProposalTimeline.vue

```diff
@@ -0,0 +1,130 @@
+<script setup lang="ts">
+import { _t } from '@/helpers/utils';
+import { NetworkID, Proposal, Space } from '@/types';
+
+type ProposalTimelineValues = {
+  network: NetworkID;
+  created?: number;
+  start: number;
+  min_end: number;
+  max_end: number;
+};
+
+type State = {
+  id: string;
+  block_number?: number;
+  value: number;
+};
+
+const props = defineProps<{ data: Proposal | Space }>();
+
+const labels = {
+  created: 'Created',
+  start: 'Start',
+  end: 'End',
+  min_end: 'Min. end',
+  max_end: 'Max. end'
+};
+
+const { getTsFromCurrent } = useMetaStore();
+
+const now = ref(parseInt((Date.now() / 1e3).toFixed()));
+
+onMounted(() => {
+  const interval = setInterval(() => {
+    now.value = parseInt((Date.now() / 1e3).toFixed());
+  }, 1e3);
```

Comment:
```suggestion
  const interval = setInterval(() => {
    now.value = parseInt((Date.now() / 1000).toFixed());
  }, 1000);
```

Let's use more readable values.

Reply:
Agree but  @bonustrack prefer this more. even this change was from him https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/components/Modal/Timeline.vue#L69 😅 all v1 have these scientific notations 

also in this repo we use it in many places 
<img width="443" alt="Untitled 2" src="https://github.com/user-attachments/assets/7a14e6e2-7159-4ae0-ba11-507e8a96431f">

not sure which way we should go

Reply:
@bonustrack are you okay if we start moving those small numbers from exponential notation to decimal?

Reply:
Sure